Training unimodal model using visual modality
visual
Epoch: 0:
Start training ...
Traceback (most recent call last):
  File "/workspace/emotion_KD250628/MKD/CREMA_D_MKD/MKD/main.py", line 506, in <module>
    main()
  File "/workspace/emotion_KD250628/MKD/CREMA_D_MKD/MKD/main.py", line 403, in main
    batch_loss, global_step = train_epoch(args, epoch, model, device,
  File "/workspace/emotion_KD250628/MKD/CREMA_D_MKD/MKD/main.py", line 77, in train_epoch
    for step, (spec, image, label) in enumerate(dataloader):
  File "/opt/conda/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 630, in __next__
    data = self._next_data()
  File "/opt/conda/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 674, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
  File "/opt/conda/lib/python3.10/site-packages/torch/utils/data/_utils/fetch.py", line 51, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/opt/conda/lib/python3.10/site-packages/torch/utils/data/_utils/fetch.py", line 51, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/workspace/emotion_KD250628/MKD/CREMA_D_MKD/MKD/dataset/CramedDataset.py", line 67, in __getitem__
    spectrogram = librosa.stft(resamples, n_fft=512, hop_length=353)
  File "/opt/conda/lib/python3.10/site-packages/librosa/core/spectrum.py", line 387, in stft
    stft_matrix[..., bl_s + off_start : bl_t + off_start] = fft.rfft(
  File "/opt/conda/lib/python3.10/site-packages/scipy/fft/_backend.py", line 28, in __ua_function__
    return fn(*args, **kwargs)
  File "/opt/conda/lib/python3.10/site-packages/scipy/fft/_basic_backend.py", line 91, in rfft
    return _execute_1D('rfft', _pocketfft.rfft, x, n=n, axis=axis, norm=norm,
  File "/opt/conda/lib/python3.10/site-packages/scipy/fft/_basic_backend.py", line 32, in _execute_1D
    return pocketfft_func(x, n=n, axis=axis, norm=norm,
  File "/opt/conda/lib/python3.10/site-packages/scipy/fft/_pocketfft/basic.py", line 61, in r2c
    return pfft.r2c(tmp, (axis,), forward, norm, None, workers)
KeyboardInterrupt
Exception ignored in atexit callback: <function _start_and_connect_service.<locals>.teardown_atexit at 0x7fa84bc7dc60>
Traceback (most recent call last):
  File "/opt/conda/lib/python3.10/site-packages/wandb/sdk/lib/service_connection.py", line 94, in teardown_atexit
    conn.teardown(hooks.exit_code)
  File "/opt/conda/lib/python3.10/site-packages/wandb/sdk/lib/service_connection.py", line 226, in teardown
    self._router.join()
  File "/opt/conda/lib/python3.10/site-packages/wandb/sdk/interface/router.py", line 75, in join
    self._thread.join()
  File "/opt/conda/lib/python3.10/threading.py", line 1096, in join
    self._wait_for_tstate_lock()
  File "/opt/conda/lib/python3.10/threading.py", line 1116, in _wait_for_tstate_lock
    if lock.acquire(block, timeout):
KeyboardInterrupt:
